src/envoy/http/authn/authenticator_base.cc

```diff
@@ -70,13 +62,27 @@ void AuthenticatorBase::validateX509(
 }
 
 void AuthenticatorBase::validateJwt(
-    const iaapi::Jwt&,
+    const iaapi::Jwt& jwt,
     const AuthenticatorBase::MethodDoneCallback& done_callback) {
   Payload payload;
   Envoy::Http::HeaderMap& header = *filter_context()->headers();
   ENVOY_LOG(debug, "{} the number of headers is {}", __func__, header.size());
 
-  bool ret = AuthnUtils::GetJWTPayloadFromHeaders(header, kJwtHeaderKey,
+  if (0 ==
```

Comment:
Use find (and keep the returned iterator for the next call) so you don't have to search the map twice (one with count, one with at)

Reply:
Done.